test/test.js

```diff
@@ -0,0 +1,32 @@
+const { electron } = require("playwright-electron");
+const assert = require("assert");
+
+const VSCODE =
```

Comment:
Could we customize this per OS? I'd love to be able to run these tests in Linux, since I'm doing 100% of my FlashCode development in Codespaces.